tests/functional/trace_latency_histogram.py

```diff
@@ -0,0 +1,126 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import time
+import pytest
+
+from api.iotrace_lat_hist_parser import LatencyHistogram
+from core.test_run import TestRun
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_param import ReadWrite, IoEngine, FioOutput
+from test_utils.size import Size, Unit
+from test_utils.os_utils import Udev
+
+
+@pytest.mark.parametrize("io_dir", [ReadWrite.write, ReadWrite.read])
+def test_latency_histogram_basic(io_dir):
+    """
+        title: Test for basic latency histogram properties
+        description: |
+            Test if samples count reported by fio equals count from iotracer (taking
+            into consideration dropped ones). Test if average.
+        pass_criteria:
+            - Fio's samples number equals number of iotracer samples + number of
+              dropped ones
+    """
+    iotrace = TestRun.plugins['iotrace']
+    read = ReadWrite.read is io_dir
+
+    for disk in TestRun.dut.disks:
```

Comment:
Ummmm that's actually leftover from copy-pasting from existing tests.
Those tests could be run on just 1 out of available disks but actually how about leaving it as it is? So that it could test all possible drives types from system (hdd, nvme...). What do you think?

Reply:
Yeah, there could be value in testing on different drives. That's fine then